src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -681,6 +681,7 @@ internal static X509Certificate2Collection BuildBagOfCerts(KeyInfoX509Data keyIn
                         }
                     }
                     catch (CryptographicException) { }
+                    catch (PlatformNotSupportedException) { }
```

Comment:
Well I guess `// Not supported on Linux` would be something. It just stands out as there's only 2 or 3 places in CoreFX where the product catches PNSE itself.